ui/espresso/FragmentScenarioSample/app/build.gradle

```diff
@@ -53,6 +51,7 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+    namespace 'com.example.android.testing.espresso.fragmentscenario'
```

Comment:
How are we determining if the `build.gradle` files need a `testNamespace` and to not abort on lint failure?

Do we need to add a `testNamespace` + `abortOnError` here for consistency? 

Also, this is a nitpick but the rest of the namespaces are capitalized, e.g. `FragmentScenario`